frame/staking/src/pallet/impls.rs

```diff
@@ -807,8 +830,8 @@ impl<T: Config> Pallet<T> {
 				// this can only happen if: 1. there a bug in the bags-list (or whatever is the
 				// sorted list) logic and the state of the two pallets is no longer compatible, or
 				// because the nominators is not decodable since they have more nomination than
-				// `T::MaxNominations`. The latter can rarely happen, and is not really an emergency
-				// or bug if it does.
+				// `T::NominationsQuota::get_quota`. The latter can rarely happen, and is not
+				// really an emergency or bug if it does.
```

Comment:
Should be `defensive!` instead of log warn.

Reply:
Done.